tests/unittests/log_filtering_functions/sdk_logging/__init__.py

```diff
@@ -0,0 +1,16 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+import logging
+
+import azure.functions as func
+
+
+sdk_logger = logging.getLogger('azure.functions')
```

Comment:
So - if someone creates a new logger using `logging.getLogger('azure.functions')` - all the logs logged would now be send to us.

Does this also send to AppInsights? I was assuming no but then I read this - 

```
Here's a list of expected behaviors:
                  local_console  customer_app_insight  functions_kusto_table
    system_log    false          true                  true
    customer_log  true           true                  false
```

Do we send all the system logs to AppInsights. I don't think so - but I'll confirm as well.

Reply:
And just doing `logging.info` would still log to App Insights. Right?

Reply:
Yeah, I will make an E2E test and see if the application insights logs can be checked efficiently. As I know, the application logs require 30 seconds delay to get the log statement.

Reply:
Just checked the application insights manually, this is a good catch. Seems like the system_log does not go to the customer_app_insight, I will amend the PR now.

![image](https://user-images.githubusercontent.com/48038149/118314292-cbb3dd80-b4a8-11eb-8577-d1b581cb849f.png)